t/mutual-ssl.t

```diff
@@ -4,12 +4,15 @@ use Test::APIcast::Blackbox 'no_plan';
 env_to_apicast(
     'APICAST_PROXY_HTTPS_CERTIFICATE' => "$Test::Nginx::Util::ServRoot/html/client.crt",
     'APICAST_PROXY_HTTPS_CERTIFICATE_KEY' => "$Test::Nginx::Util::ServRoot/html/client.key",
+    'APICAST_PROXY_HTTPS_PASSWORD_FILE' => "$Test::Nginx::Util::ServRoot/html/passwords.file",
+    'APICAST_PROXY_HTTPS_SESSION_REUSE' => 'on',
 );
 
 run_tests();
 
 __DATA__
-=== TEST 1: mutual SSL
+
+=== TEST 1: Mutual SSL with password file
```

Comment:
Maybe adding a new test would have been better so we can be sure that it works with and without the password file.